spec/accounting_money_parser_spec.rb

```diff
@@ -7,95 +7,95 @@
     end
     
     it "should parse parenthesis as a negative amount eg (99.00)" do
```

Comment:
`s/should be/is/g`
`s/should not be/is not/g`
`s/should not/does not/g`
`s/should (\w+)/$1s/g`